src/functions/pollDelegatedBookings/domain/delegated-bookings-cache-reconciler.ts

```diff
@@ -3,21 +3,27 @@ import {
   cacheDelegatedBookingDetails,
 } from '../framework/repo/dynamodb/cached-delegated-bookings-repository';
 import { DelegatedBookingDetail } from '../../../common/application/models/delegated-booking-details';
+import { DateTime } from '../../../common/application/utils/date-time';
+import { decompressDelegatedBooking } from '../application/booking-compressor';
+
+const NUMBER_OF_DAYS_TO_RETAIN_CACHED_BOOKINGS = 60;
```

Comment:
maybe this could come from an environment variable so that we don't have to upgrade the code when it changes?

Reply:
Ideally yes, however it's not possible due to this work being quite time-sensitive.

Reply:
Alright, it's pretty easy to change the constant anyway